docs/source-pytorch/cli/lightning_cli_advanced_3.rst

```diff
@@ -176,10 +177,13 @@ module and data module.
 Models with multiple submodules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Many use cases require to have several modules each with its own configurable options. One possible way to handle this
-with LightningCLI is to implement a single module having as init parameters each of the submodules. Since the init
-parameters have as type a class, then in the configuration these would be specified with :code:`class_path` and
-:code:`init_args` entries. For instance a model could be implemented as:
+Many use cases require to have several modules, each with its own configurable options. One possible way to handle this
+with ``LightningCLI`` is to implement a single module having as init parameters each of the submodules. This is known as
+`dependency injection <https://en.wikipedia.org/wiki/Dependency_injection>`__ which is a good approach to improve
+decoupling in your code base.
+
+Since the init parameters of the model have as type hint a class, in the configuration these would be specified with
+``class_path`` and ``init_args`` entries. For instance a model could be implemented as:
```

Comment:
```suggestion
Many use cases require to have several modules, each with its own configurable options. One possible way to handle this
with ``LightningCLI`` is to implement a single module having as init parameters each of the submodules. This is known as
`dependency injection <https://en.wikipedia.org/wiki/Dependency_injection>`__ which is a good approach to improve
decoupling in your code base.

Since the init parameters of the model have as a type hint a class, in the configuration, these would be specified with
``class_path`` and ``init_args`` entries. For instance, a model could be implemented as:
```